Blockzilla/SettingsViewController.swift

```diff
@@ -120,7 +122,18 @@ class SettingsViewController: UIViewController, UITableViewDataSource, UITableVi
         view.backgroundColor = UIConstants.colors.background
 
         title = UIConstants.strings.settingsTitle
-
+        
+        // Add Face ID or Touch ID toggle as appropriate based on the device
+        if context.canEvaluatePolicy(.deviceOwnerAuthenticationWithBiometrics, error: &biometricError) {
+            if context.biometryType == .faceID {
+               toggles.insert( BlockerToggle(label: UIConstants.strings.labelFaceIDLogin, setting: SettingsToggle.biometricLogin, subtitle: UIConstants.strings.labelFaceIDLoginDescription), at: 1)
```

Comment:
nit: remove space before func arg

Reply:
consider this to shorten the code in this line and indicate that toggles.insert doesn't vary in this logic:
``` let toggle: BlockerToggle; if ... { toggle = BlockerToggle(...) } else if { toggle = BlockerToggle(...) }; toggles.insert(toggle, at: 1)```